components/brave_wallet_ui/components/desktop/views/portfolio/portfolio-overview.tsx

```diff
@@ -194,24 +222,62 @@ export const PortfolioOverview = () => {
     const grandTotal = visibleAssetFiatBalances.reduce(function (a, b) {
       return a.plus(b)
     })
-    return grandTotal.formatAsFiat(defaultCurrencies.fiat)
+    return grandTotal.format()
```

Comment:
Let's do `return grandTotal` so calculations are easier thanks to `Amount` class.

Reply:
Now returns an `Amount`